.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: /usr/src/Slicer-LiverAnalysis
+    docker:
+      - image: olevs/slicer-alive-build
+    steps:
+      - checkout
+      - run:
+          name: Update 3D Slicer
+          command: |
+            cd /usr/src/Slicer
+            git checkout .
+            git pull
+            
+      # Rebuilding Slicer may take a long time
+      # For now, just use the build files already present in the Docker image
+      #- run:
+      #    name: Build 3D Slicer
+      #    environment:
+      #      BUILD_TOOL_FLAGS: "-j3"
+      #    command: |
+      #      ../Slicer-build/BuildSlicer.sh
+      
+      # Not needed, as this is done by the checkout step
+      #- run:
+      #    name: Update LiverAnalysis plugin
+      #    command: |
+      #      cd /usr/src/Slicer-LiverAnalysis
+      #      git checkout .
+      #      git pull
+      
+      - run:
+          name: Build LiverAnalysis plugin
+          environment:
+            BUILD_TOOL_FLAGS: "-j3"
```

Comment:
I would leave this as `-j1` just to start. I think extensions will not take much to compile and we will probably get only one core with the free plan anyways

Reply:
I set it to -j1 for now. It probably don't matter much. Slicer uses -j3.